src/styles.ts

```diff
@@ -37,14 +37,14 @@ export const styles = css`
 		);
 		--contour-color: var(--line-color);
 		--contour-size: var(--cosmoz-input-contour-size);
+		--border-color: var(--contour-color);
```

Comment:
You define --border-color, but you are not using it anywhere. What is the purpose of defining this var?